### PR TITLE
[wip] fix koka language server

### PR DESCRIPTION
**Pull request by MangoIV**
This is still in draft because I couldn't make the language server attach. 

Currently I have 

```lua
vim.filetype.add({
  extension = {
    kk = "koka",
  },
})
```

in my `init.lua` but that doesn't seem to suffice. 

Help would be greatly appreciated.

### CODE REVIEW

**Comment by MangoIV** [comment]
I have confirmed with the original author of the configuration, that the koka plugin never worked.

**Comment by MangoIV** [comment]
ahhhh now I know why.

**Comment by MangoIV** [comment]
image

File: lua/lspconfig/server_configurations/koka.lua [hunk 1]
```diff
@@ -1,36 +1,23 @@
...
-}
-
 return {
   default_config = {
-    cmd = { 'koka', '--language-server' },
-    filetypes = { 'kk' },
+    cmd = { 'koka', '--language-server', '--lsstdio' },
+    filetypes = { 'koka' },
+    single_file_support = true,
     root_dir = function(fname)
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/koka.lua:8]:
```suggestion
    root_dir = util.find_git_ancestor,
```

**Comment by MangoIV** [comment]
this commit lint thing is mildly annoying, why did the same mesage pass only a few minutes ago?

**Comment by MangoIV** [comment]
oh I fucked up rebasing.

**Comment by MangoIV** [comment]
@glepnir am I expected to fix this linter error? it's in a file that I didn't even touch.

**Comment by glepnir** [comment]
Nope not your edit i will fix later.

**Pull request merged**